Source/WTF/wtf/DebugHeap.h

```diff
@@ -88,7 +90,9 @@ class DebugHeap {
 
 #else // ENABLE(MALLOC_HEAP_BREAKDOWN)
 
-#define DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(Type) \
+#define DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(Type) DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER_AND_EXPORT(Type, nullptr)
```

Comment:
`nullptr` is weird, since this parameter is never used like a pointer. I think I would just put WTF_EXPORT_PRIVATE here? (That would also enable you to hoist this definition outside of the `#ifdef` and only define it once, if you'd like.)